price-feeder/CHANGELOG.md

```diff
@@ -49,6 +49,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 - [#648](https://github.com/umee-network/umee/pull/648) Add Coinbase as a provider.
 
+### Bug Fixes
+
+- [#675](https://github.com/umee-network/umee/pull/675) Add verification for subscribe currency pairs with empty parameter.
```

Comment:
Forgot to mention, I would also update this to say : 
```
- [#675](https://github.com/umee-network/umee/pull/675) Add necessary input validation to SubscribePairs in the price feeder.
```

Reply:
agreed, thanks!

Reply:
done at https://github.com/umee-network/umee/pull/675/commits/84374f0388352ba1db1bb6ae95cca967d0604ff0